src/media/css/app-list.styl

```diff
@@ -27,11 +27,27 @@ $desktop-tile-padding = 20px 50px;
     list-style: none;
     margin-top: 30px;
 
-    li:not(.loadmore) {
+    .app-list-app {
```

Comment:
can you add this to the feed detail pages (and everywhere else we have an app-list) as well since this change is introduced in this patch?